apps/omg_watcher/lib/db/transaction.ex

```diff
@@ -21,11 +21,11 @@ defmodule OMG.Watcher.DB.Transaction do
 
   alias OMG.API.State.Transaction
   alias OMG.API.Utxo
-  alias OMG.Watcher.DB
+  alias OMG.Watcher.{DB, DB.TxOutput}
```

Comment:
I've seen a lot changes actually expanding this kind into separate lines - it's more alchemist ;)